scripts/linux/electron-installer-appimage.sh

```diff
@@ -143,12 +144,19 @@ else
 fi
 
 # Generate AppImage
+mkdir -p "$(dirname "$ARGV_OUTPUT")"
 rm -f "$ARGV_OUTPUT"
 
 APPIMAGEASSISTANT_PATH=/tmp/AppImageAssistant.AppImage
 download_executable \
   "$APPIMAGES_GITHUB_RELEASE_BASE_URL/AppImageAssistant_$APPIMAGES_TAG-$APPIMAGES_ARCHITECTURE.AppImage" \
   $APPIMAGEASSISTANT_PATH
 
-$APPIMAGEASSISTANT_PATH "$APPDIR_PATH" "$ARGV_OUTPUT"
+$APPIMAGEASSISTANT_PATH "$APPDIR_PATH" "$(dirname "$ARGV_OUTPUT")/$OUTPUT_FILENAME"
```

Comment:
They way this script works is that it generates an AppImage (which is located at `$OUTPUT_FILENAME`), it creates a ZIP containing the AppImage, and then removes the standalone AppImage. Thus, the AppImage (`$OUTPUT_FILENAME`) can be thought of as a "temporary file". I agree that putting it as a sibling of the expected output path looks a bit awkward. What about putting it in `$TMPDIR` instead?